internal/cli/cmd/create/role.go

```diff
@@ -145,6 +154,37 @@ func (o *createRoleOptions) run(ctx context.Context) error {
 		return fmt.Errorf("get client from config: %w", err)
 	}
 
+	claims := []rbacapi.Claim{}
+
+	if o.Subs != nil {
+		claims = append(claims, rbacapi.Claim{
+			Name:   "subs",
+			Values: o.Subs,
+		})
+	}
+
+	if o.Groups != nil {
+		claims = append(claims, rbacapi.Claim{
+			Name:   "groups",
+			Values: o.Groups,
+		})
+	}
+
+	if o.Emails != nil {
+		claims = append(claims, rbacapi.Claim{
+			Name:   "emails",
+			Values: o.Emails,
+		})
+	}
+
+	for _, claimFlagValue := range o.Claims {
+		claimFlagNameAndValue := strings.Split(claimFlagValue, "=")
```

Comment:
We should probably verify that the split results in exactly two tokens.

My best advice is to test for the presence of exactly one "=" in the `validate()` function.

Reply:
Yep makes sense, I've added that to the validate functions.